go.mod

```diff
@@ -21,8 +21,9 @@ require (
 	github.com/frankban/quicktest v1.11.3 // indirect
 	github.com/fsouza/go-dockerclient v1.7.0
 	github.com/go-kit/kit v0.9.0
-	github.com/golang/protobuf v1.3.3
+	github.com/golang/protobuf v1.5.2
```

Comment:
@guoger @Param-S I thought etcd required the older protobuf, and therefore we were stuck at the older version of protobuf and ginkgo until the etcd update is complete. Right?

Reply:
Yes, It is correct. It is recorded here: https://github.com/hyperledger/fabric/pull/2185#issuecomment-737650631.

Reply:
One more issue for reference: https://jira.hyperledger.org/browse/FAB-18363